### PR TITLE
[PEP8] Remove some unused variables and linter comments.

### PR DESCRIPTION
**Pull request by jensrudolf**
This commit introduces some small changes to avoid instances of unused
variables. There are some more places in the code which might be cleaned
up by a later commit.

### CODE REVIEW

File: src/main/python/linuxband/gui/events/groove.py [hunk 1]
```diff
@@ -39,17 +39,17 @@ def __init__(self, glade, grooves):
 
     def on_treeview1_cursor_changed_callback(self, treeview):
         """ User clicked on the groove in the first column. """
-        path, column = treeview.get_cursor() #@UnusedVariable
+        path = treeview.get_cursor()[0]
         gr = self.__groovesModel[path[0]]
```
Comment (noseka1) [inline src/main/python/linuxband/gui/events/groove.py]: This change looks suspicious. Line 43 is still using index on the path variable.

File: src/main/python/linuxband/gui/events/groove.py [hunk 2]
```diff
@@ -39,17 +39,17 @@ def __init__(self, glade, grooves):
...
         self.__treeview2.set_model(gr[6])
         self.__update_groove_info(gr)
         self.__update_groove_event(gr[0])
 
     def on_treeview2_cursor_changed_callback(self, treeview):
         """ User clicked on the variation of groove. """
-        path, column = self.__treeview1.get_cursor() #@UnusedVariable
+        path = self.__treeview1.get_cursor()[0]
         model = self.__groovesModel[path[0]][6]
```
Comment (noseka1) [inline src/main/python/linuxband/gui/events/groove.py]: Similar problem. The index needs to go I think.

File: src/main/python/linuxband/gui/events/groove.py [hunk 3]
```diff
@@ -39,17 +39,17 @@ def __init__(self, glade, grooves):
...
-        path, column = self.__treeview2.get_cursor() #@UnusedVariable
+        path = self.__treeview2.get_cursor()[0]
         gr = model[path[0]]
```
Comment (noseka1) [inline src/main/python/linuxband/gui/events/groove.py]: And one more occurrence.

File: src/main/python/linuxband/gui/events/groove.py (hunk 1 above)
Comment (jensrudolf) [reply src/main/python/linuxband/gui/events/groove.py]: According to the [Treeview documentation](https://developer.gnome.org/pygtk/stable/class-gtktreeview.html#method-gtktreeview--get-cursor) get_cursor() returns a tuple as (path, column) where path is a TreePath object, which itself is always a tuple as stated in the [TreeModel Interface docs](https://docs.huihoo.com/pygtk/2.0-tutorial/sec-TreeModelInterface.html). So accessing path[0] seems perfectly valid to me.

Comment (noseka1) [reply src/main/python/linuxband/gui/events/groove.py]: Thanks for the info. Works great.

**Pull request merged**